src/hooks/useClickableProps/__stories__/useClickableProps.stories.mdx

```diff
@@ -0,0 +1,275 @@
+import { useCallback, useRef } from "react";
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Link, Tip } from "../../../storybook";
+import useClickableProps from "../useClickableProps";
+import "./useClickableProps.stories.scss";
+
+<Meta title="Hooks/useClickableProps" />
+
+# useClickableProps
```

Comment:
Also worth to add the following tips in the Clickable component:
Tip
If you'd like to set clickable functionality on a specific element inside your React component instead of using a wrapper, you can also look in our useClickableProps hook.